network-core/src/server/transaction.rs

```diff
@@ -12,6 +13,10 @@ use std::{error, fmt};
 /// Interface for the blockchain node service implementation responsible for
 /// validating and accepting transactions.
 pub trait TransactionService {
+
+    /// Transaction in the blockchain.
+    type Transaction: property::Transaction;
```

Comment:
Alternatively, import `Transaction`, it does not conflict with associated type names.

Reply:
@qnikst TIL that you can also do `use property::Transaction as _;` go get the trait methods without importing the trait name.

Reply:
Only since [rust 1.33](https://blog.rust-lang.org/2019/02/28/Rust-1.33.0.html#import-as-_)